.gitignore

```diff
@@ -1 +1,2 @@
 _site
+.DS_Store
```

Comment:
And this line change should prbably be in a separate commit :) can you update the commit? You can do so by run `git rebase --interactive master`.